packages/core/src/transforms/resize.ts

```diff
@@ -4,12 +4,33 @@ import { getPosition } from './position'
 import { getKernel } from './kernel'
 import { getBackground } from './background'
 import { setMetadata, getMetadata } from "../lib/metadata";
+import { Sharp } from "sharp";
```

Comment:
This import seems to be unnecessary 😄

Reply:
Hah, good catch!  I started with a helper that took in the Sharp image, removed the helper but missed this type import